### PR TITLE
Add omise-node document

### PR DESCRIPTION
**Pull request by zdk**
Add a document for Omise NodeJS client library.

### CODE REVIEW

File: source/api/customers/update/_update_simple.sh [hunk 1]
```diff
@@ -1,5 +1,3 @@
-curl https://api.omise.co/customers/cust_test_4xtrb759599jsxlhkrb \
-  -X PATCH \
-  -u skey_test_4xsjvwfnvb2g0l81sjz: \
-  -d "email=john.smith@example.com" \
-  -d "description=Another description"
```
Comment (robinclart) [inline source/api/customers/update/_update_simple.sh]: You overwrite on the wrong file it seems.

File: source/api/refunds/create/_create.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+omise.charges.createRefund('chrg_test_4ype2jynk2len88i4er', 
+{ 'amount': 100000 }, function(error, refund) {});
```
Comment (robinclart) [inline source/api/refunds/create/_create.js]: Please add an empty newline to all files.

**Comment by robinclart** [comment]
Adding node to the example panels is done through two files:
- first `data/langs.json` files. The key is the name, the second is the file extension.
- then in `locales/en.yml` under `langs:`.

File: source/api/customers/update/_update_simple.sh (hunk 1 above)
Comment (zdk) [inline source/api/customers/update/_update_simple.sh]: oh, crap. sorry, fixing soon.

File: source/api/refunds/create/_create.js (hunk 2 above)
Comment (zdk) [inline source/api/refunds/create/_create.js]: kk

**Comment by zdk** [comment]
Thanks, I'll add to both files.

**Comment by robinclart** [comment]
:+1:

**Pull request merged**